docs/resources/dms_kafka_instance.md

```diff
@@ -0,0 +1,221 @@
+
+* `available_zones` - (Required, List, ForceNew) Specifies the ID of an AZ. The parameter value can not be
+    left blank or an empty array. Changing this creates a new instance resource.
+
+* `product_id` - (Required, String, ForceNew) Specifies a product ID. Changing this creates a new instance resource.
+
+* `kafka_manager_user` - (Required, String, ForceNew) Specifies the username for logging in to the Kafka Manager.
+    The username consists of 4 to 64 characters and can contain letters, digits, hyphens (-), and underscores (_).
+    Changing this creates a new instance resource.
+
+* `kafka_manager_password` - (Required, String, ForceNew) Specifies the password for logging in to the Kafka Manager.
+    The password must meet the following complexity requirements: Must be 8 to 32 characters long.
+    Must contain at least 2 of the following character types: lowercase letters, uppercase
+	letters, digits, and special characters (`~!@#$%^&*()-_=+\|[{}]:'",<.>/?).
+    Changing this creates a new instance resource.
+
+* `maintain_begin` - (Optional, String) Specifies the time at which a maintenance time window starts.
+    Format: HH:mm:ss.
+    The start time and end time of a maintenance time window must indicate the time segment of
+	a supported maintenance time window. The start time must be set to 22:00, 02:00, 06:00, 10:00,
+    14:00, or 18:00. Parameters `maintain_begin` and `maintain_end` must be set in pairs. If
+    parameter `maintain_begin` is left blank, parameter `maintain_end` is also blank. In this case, 
+    the system automatically allocates the default start time 02:00.
+
+* `maintain_end` - (Optional, String) Specifies the time at which a maintenance time window ends.
+    Format: HH:mm:ss.
+    The start time and end time of a maintenance time window must indicate the time segment of
+	a supported maintenance time window. The end time is four hours later than the start time.
+    For example, if the start time is 22:00, the end time is 02:00. Parameters `maintain_begin`
+    and `maintain_end` must be set in pairs. If parameter `maintain_end` is left blank, parameter
+    `maintain_begin` is also blank. In this case, the system automatically allocates the default
+    end time 06:00.
+
+* `public_ip_ids` - (Optional, List, ForceNew) Specifies the IDs of the elastic IP address (EIP)
+    bound to the DMS kafka instance. The num of IDs needed ranges:
+    - When specification is 100MB: 3
+    - When specification is 300MB: 3
+    - When specification is 600MB: 4
+    - When specification is 1200MB: 8
+
+    Changing this creates a new instance resource.
+
+* `retention_policy` - (Optional, String) Specifies the action to be taken when the memory usage reaches
+    the disk capacity threshold. Value range:
+    - `time_base`: Automatically delete the earliest messages.
+    - `produce_reject`: Stop producing new messages.
+
+* `connector_enable` - (Optional, Bool, ForceNew) Specifies whether to enable dumping.
```

Comment:
suggest rename it to `dumping` to keep with the console

Reply:
fixed